lib/commonConfig.js

```diff
@@ -65,6 +65,7 @@ function processEnvironmentVariables() {
              'IOTA_AUTH_PORT',
              'IOTA_AUTH_USER',
              'IOTA_AUTH_PASSWORD',
+             'IOTA_AUTH_PERMANENT_TOKEN',
```

Comment:
I understand the new configuration should be described at:

* https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/installationguide.md#global-configuration
* https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/installationguide.md#configuration-using-environment-variables

Reply:
This was already done for https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/installationguide.md#global-configuration as you can see in https://github.com/telefonicaid/iotagent-node-lib/pull/664/files#diff-9b0a554150d1aeae259b560098da86f1

Reply:
Fixed with a50cc84a4aaa271e8dc09919c2c60d8b13be1e97

Reply:
Commit a50cc84 includes the new env var in the env vars table but not the explanation of the new parameter (authentication.permanentToken), which is typically done in the just above section "Global Configuration".

Or maybe I'm missing something?

Reply:
This was done in https://github.com/telefonicaid/iotagent-node-lib/pull/664/commits/6efbf2f5b4df016d47f1ef6088765f4e07f20176

Reply:
Ok, here is the text:

> In order to use this type of token, an additional parameter `permanentToken` must be set to `true` in the `authentication` configuration. An environment variable `IOTA_AUTH_PERMANENT_TOKEN` can be also used for the same purpose

I missed it in the dense block of text in which it appears :)

NTC